src/java.desktop/share/classes/sun/swing/FilePane.java

```diff
@@ -1209,38 +1209,22 @@ public Component getTableCellRendererComponent(JTable table, Object value,
 
                 if (listViewWindowsStyle) {
                     updateMessageFormatPattern(kiloByteString);
-                    if (len < 1000L) {
-                        displayedFileSize[0] = ((len +99) / 100L) / 10.0d;
-                    } else {
-                        displayedFileSize[0] = roundToOneDecimalPlace(len);
-                    }
+                    displayedFileSize[0] = roundToOneDecimalPlace(len);
```

Comment:
Does it correctly display the file sizes when `listViewWindowsStyle` is `true`?

Reply:
Yeah, it will show file size is KB.